packages/config-plugins/src/utils/Updates.ts

```diff
@@ -1,14 +1,30 @@
 import { Android, ExpoConfig, IOS } from '@expo/config-types';
 import { getRuntimeVersionForSDKVersion } from '@expo/sdk-runtime-versions';
+import fs from 'fs';
 import { boolish } from 'getenv';
+import path from 'path';
 
 import { AndroidConfig, IOSConfig } from '..';
+import { resolvePackageRootFolder } from './resolvePackageRootFolder';
 
 export type ExpoConfigUpdates = Pick<
   ExpoConfig,
   'sdkVersion' | 'owner' | 'runtimeVersion' | 'updates' | 'slug'
 >;
 
+export function getExpoUpdatesPackageVersion(projectRoot: string): string | null {
+  const expoUpdatesRootFolder = resolvePackageRootFolder(projectRoot, 'expo-updates');
```

Comment:
`resolvePackageRootFolder` will be deleted.

```suggestion
  let expoUpdatesRootFolder = resolveFrom.silent(projectRoot, 'expo-updates/package.json');
  if (expoUpdatesRootFolder) expoUpdatesRootFolder = path.dirname(expoUpdatesRootFolder);
```